homeassistant/const.py

```diff
@@ -323,6 +323,7 @@ class Platform(StrEnum):
 STATE_OPENING: Final = "opening"
 STATE_CLOSED: Final = "closed"
 STATE_CLOSING: Final = "closing"
+STATE_BUFFERING: Final = "buffering"
```

Comment:
media player could use a state enum.

Reply:
Agreed, but lets keep it out of scope for this PR :)